wls-gui-wahllokalsystem/src/api/fetch-utils.ts

```diff
@@ -143,3 +173,13 @@ function getXSRFToken(): string {
   );
   return (help ? help.pop() : "") as string;
 }
+
+function isWLSException(obj: any): obj is WLSException {
```

Comment:
ich finde die Funktion besser in `WlsException` aufgehoben

Reply:
siehe [c288128](https://github.com/it-at-m/Wahllokalsystem/pull/633/commits/c288128c66df0cb8a67fe13d3aa31efc60ac7b0a)